### PR TITLE
Rename chrysalis docs

### PR DESCRIPTION
**Pull request by Dr-Electron**
# Description of Change

As this docs in the future house introduction docs for all kinds of updates I renamed it.

Can/Should we already rename the repo too?
Then I need to make 2 more changes

## Type of Change

- Documentation Enhancement

## How the Change Has Been Tested

Not tested. Let's wait for the action to do that :trollface: 

## Change Checklist

- [x] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have tested the docs locally

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,6 @@
 
 > Everything about Chrysalis for developers.
 
-You can see a rendered version on [https://wiki.iota.org/chrysalis-docs/welcome](https://wiki.iota.org/chrysalis-docs/welcome).
+You can see a rendered version on [./welcome](./welcome).
```
Comment (Dr-Electron) [inline README.md:5]:
```suggestion
You can see a rendered version on [https://wiki.iota.org/introduction/welcome](./welcome).
```

**Comment by lucas-tortora** [comment]
Looks good to me :) 

Since the Chrysalis docs are referenced a lot in other repos, I think we may want to consider leaving this as a draft while we create link update PRs in all the other repos and then merge once everything is approved. Wdyt?

**Review by lucas-tortora — APPROVED** [review]

**Comment by Dr-Electron** [comment]
> Looks good to me :)
> 
> Since the Chrysalis docs are referenced a lot in other repos, I think we may want to consider leaving this as a draft while we create link update PRs in all the other repos and then merge once everything is approved. Wdyt?

We can just merge. I talked with DevOps and they will just create a redirection. Then we can fix the other things too and also announce it. Maybe something else like the website is using it

**Comment by Dr-Electron** [comment]
What do you think about renaming the repo? Shouldn't be a problem. Github creates redirect itself afaik just in case

**Comment by lucas-tortora** [comment]
> What do you think about renaming the repo? Shouldn't be a problem. Github creates redirect itself afaik just in case

I can't say I remember ever doing this, but [according to GH](https://github.community/t/how-long-does-github-forward-renamed-moved-repos/582) it should be OK

**Comment by Dr-Electron** [comment]
> > What do you think about renaming the repo? Shouldn't be a problem. Github creates redirect itself afaik just in case
> 
> I can't say I remember ever doing this, but [according to GH](https://github.community/t/how-long-does-github-forward-renamed-moved-repos/582) it should be OK, unless we create another `chrysalis-docs` repo in the same org.

With the release of stardust I don't see that happening anymore :trollface:

**Review by jlvandenhout — APPROVED** [review]

**Pull request merged**